charts/generic-service/values.yaml

```diff
@@ -87,6 +87,13 @@ scheduling:
   nodePreferences: {}
   nodeExclusions: {}
   tolerations: []
+    ## Example:
+    ## tolerations: ["node.deepl.dev/k8s-kubevirt"]
+    ## Or you can add more complex tolerations as objects here
+    ## tolerations:
+    ## - key: "node.deepl.dev/k8s-gpu-worker"
+    ##   effect: NoSchedule
+    ##   value: "true"
```

Comment:
We don't have examples for any of the other values in `values.yaml` either.

```suggestion
```

Perhaps we can put this in the `README.md` instead?

Perhaps something like:

```md
| `scheduling.tolerations` | `[]`  | Taints on nodes this service can tolerate as strings or complex tolerations as objects |
```